devenv.nix

```diff
@@ -42,14 +42,18 @@ in
 
   languages.rust = {
     enable = true;
-    channel = "beta";
+    channel = "stable";
     components = [ "rustc" "cargo" "clippy" "rustfmt" "rust-analyzer" ];
   };
 
   languages.python = {
     enable = true;
     poetry = {
       enable = true;
+      activate.enable = true;
+      # normal package is 1.71.0 which seems to break on arm macos
+      # this installs 1.4.2 instead
+      package = pkgs-stable.poetry;
```

Comment:
Is that the last working version? It looks old.

Reply:
https://github.com/NixOS/nixpkgs/pull/281397

it was fixed 11 hours ago so added that as the source

Reply:
(now v1.7.1)